packages/webpack/mixins/build/mixin.core.js

```diff
@@ -81,13 +83,41 @@ class WebpackBuildMixin extends Mixin {
       })
     );
   }
-  diagnose({ diagnoseDuplicatePackages, diagnoseInvalidPackages }) {
-    diagnoseDuplicatePackages('webpack');
-    diagnoseInvalidPackages(__dirname, '@babel/polyfill');
-    diagnoseInvalidPackages(
-      dirname(require.resolve('@babel/polyfill/package.json')),
-      'core-js'
-    );
+  checkHoistedVersion(context, packageName) {
```

Comment:
Let us please not introduce non-hook methods to mixins - see [docs](https://github.com/untool/untool/blob/master/packages/core/README.md#api):

> While it is technically possible to define non-mixin utility methods on your mixin, doing so is strongly discouraged.